app/src/main/java/org/sopt/and/data/di/NetworkModule.kt

```diff
@@ -0,0 +1,73 @@
+package org.sopt.and.data.di
+
+import android.util.Log
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.sopt.and.BuildConfig
+import org.sopt.and.sharedpreference.User
+import retrofit2.Retrofit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object NetworkModule {
+
+    private const val CONTENT_TYPE = "application/json"
+    private const val BASE_URL = BuildConfig.BASE_URL
+
+    @Singleton
+    @Provides
+    fun provideJson(): Json {
+        return Json {
+            ignoreUnknownKeys = true
+            isLenient = true
+            encodeDefaults = true
+        }
+    }
+
+    @Singleton
+    @Provides
+    fun provideLoggingInterceptor(): HttpLoggingInterceptor {
+        return HttpLoggingInterceptor { message ->
+            Log.d("Retrofit2", "CONNECTION INFO -> $message")
+        }.apply {
+            level = HttpLoggingInterceptor.Level.BODY
+        }
+    }
+
+
+    @Singleton
+    @Provides
+    fun provideAuthInterceptor(user: User): AuthInterceptor {
+        return AuthInterceptor(user)
+    }
+
+    @Singleton
+    @Provides
+    fun provideOkHttpClient(
+        loggingInterceptor: HttpLoggingInterceptor,
+        authInterceptor: AuthInterceptor
+    ): OkHttpClient {
+        return OkHttpClient.Builder()
+            .addInterceptor(loggingInterceptor)
+            .addInterceptor(authInterceptor)
+            .build()
```

Comment:
릴리즈에는 필요없는 interceptor이므로 디버그 앱에만 추가해주심이 어떨까요
```suggestion
        val builder = OkHttpClient.Builder()
        if (BuildConfig.DEBUG) builder.addInterceptor(loggingInterceptor)
        builder.addInterceptor(authInterceptor)
        return builder.build()
```

Reply:
꼼꼼하게 봐주셔서 감사합니다! 릴리즈 디버그 구분해서 사용할 생각은 못했네요!